### PR TITLE
Added super basic WLED plugin

### PR DESCRIPTION
**Pull request by billyjacoby**
Totally open to new and better ways of doing this, but this has seemed to work for me for a few weeks now!

### CODE REVIEW

File: IoT/wled.1m.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#!/usr/local/bin/node
+//  -*- coding: utf-8 -*-xbar
+//  <xbar.title>WLED</xbar.title>
+//  <xbar.version>v1.0</xbar.version>
+//  <xbar.author>Billy Jacoby</xbar.author>
+//  <xbar.author.github>billyjacoby</xbar.author.github>
+//  <xbar.desc>Show and manage WLED devices from the menu bar.</xbar.desc>
+//  <xbar.dependencies>node,wled,axios</xbar.dependencies>
```
Comment (matryer) [inline IoT/wled.1m.js:8]: Please add an image of what the plugin looks like when installed.

File: IoT/wled.1m.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+//  Please install the npm dependencies in a folder named "wled" - I'm open to better ways of doing this too...
+
+// <xbar.var>string(VAR_HOST=""): WLED IP Address or hostname</xbar.var>
+const axios = require("./wled/node_modules/axios");
+const bitbar = require("./wled/node_modules/bitbar");
```
Comment (matryer) [inline IoT/wled.1m.js:14]: maybe this should be `xbar` now?

**Review by matryer — CHANGES_REQUESTED** [review]
Nice job.

File: IoT/wled.1m.js (hunk 2 above)
Comment (billyjacoby) [reply IoT/wled.1m.js:14]: The NPM package seems to have not been renamed yet - i think it works the same way but there's no `xbar` package.

The documentation for NodeJS plugins still recommends using the ```bitbar``` plugin too.

File: IoT/wled.1m.js (hunk 1 above)
Comment (billyjacoby) [reply IoT/wled.1m.js:8]: Fixed in most recent commit!

**Comment by billyjacoby** [comment]
Is this good to go?

**Comment by billyjacoby** [comment]
Is there anything else I can do to get this merged?

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**